scene/2d/physics_body_2d.h

```diff
@@ -317,9 +345,18 @@ class CharacterBody2D : public PhysicsBody2D {
 	real_t get_floor_snap_length();
 	void set_floor_snap_length(real_t p_floor_snap_length);
 
+	real_t get_free_mode_min_slide_angle() const;
+	void set_free_mode_min_slide_angle(real_t p_radians);
+
 	uint32_t get_moving_platform_ignore_layers() const;
 	void set_moving_platform_ignore_layers(const uint32_t p_exclude_layer);
 
+	void set_motion_mode(MotionMode p_mode);
+	MotionMode get_motion_mode() const;
+
+	void _move_and_slide_top_down_logic(Vector2 &motion);
+	void _move_and_slide_side_logic(Vector2 &motion, const Vector2 &prev_platform_velocity, const Vector2 &prev_floor_normal, const RID &prev_platform_rid, int platform_layer, bool was_on_floor);
```

Comment:
Missing `p_` prefix for arguments in both methods.

Also to make things a bit cleaner, I would rather pass `const Vector2 &p_motion` as an argument, and make a local copy of motion within the method for local modifications, rather than passing a non-const reference which is not used after the method call.

And `RID` doesn't really need to be passed by reference, since it's just the same as an integer type.